source/installation_guide/upgrading/from-2.3-to-2.3.x.rst

```diff
@@ -22,3 +22,5 @@ Upgrading Dovecot v2.3.x to v2.3.15
 ===================================
 
  * The 'SNIPPET' and 'PREVIEW (w/explicit algorithm selection)' IMAP commands have been deprecated. The new RFC 8970 compliant PREVIEW command should be exclusoively used in the future.
+ * Compression levels now accept per-algorithm value.
+ * :ref:`plugin-compress` now uses per-algorithm compression levels, the old setting is ignored.
```

Comment:
What's the difference between these two? Also I'd clarify to maybe:

 * :ref:`plugin-compress` now uses per-algorithm compression level settings. The old setting is ignored.